common/containers/Tabs/ENS/components/NameInput/components/HOC.jsx

```diff
@@ -0,0 +1,45 @@
+// @flow
+import * as React from 'react';
```

Comment:
You can just `import React from 'react';`, no need for `* as`

Reply:
Agreed